src/components/identity/logo.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Link } from "gatsby";
+
+const Logo = ({ className, ...props }) => (
+  <Link to="/" className="su-block su-w-fit" {...props}>
```

Comment:
```suggestion
  <Link to="/" className="logo su-block su-w-fit" {...props}>
```

You've been adding names to others, why not this one?

Reply:
Good point - adding these classes might help with Algolia search too